src/board.h

```diff
@@ -0,0 +1,90 @@
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "block.h"
+
+typedef unsigned int uint;
+
+enum MoveDirection
+{
+    LEFT,
+    DOWN,
+    RIGHT,
+    UP
+};
+
+extern constexpr char moveRow[4] = {0, +1, 0, -1};
+extern constexpr char moveCol[4] = {-1, 0, +1, 0};    // imo nie ma sensu tego trzymać w strukturze pozycji
```

Comment:
Niech będzie, na razie usuń ten komentarz i btw. to ma być const i nazwy zmiennych drukowane z podłogami: MOVE_ROWS, MOVE_COLS

Reply:
I nie wiem czy wgl wiesz co oznacza extern xd

Reply:
to extern też wywalić?